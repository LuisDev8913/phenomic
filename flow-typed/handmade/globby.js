declare module "globby" {
  declare type pattern = string | Array<string>;
  declare type async = (
    pattern: pattern,
    options: Object
  ) => Promise<Array<string>>;
  declare module.exports: {
    (): async,
    sync: (pattern: pattern, options: Object) => Array<string>
  };
}
