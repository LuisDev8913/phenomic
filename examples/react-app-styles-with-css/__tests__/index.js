// @flow

import fs from "fs";
import path from "path";

/* eslint-disable import/no-extraneous-dependencies */
import globby from "globby";

it("should build example correctly", () => {
  const testFolder = __dirname + "/../dist";
  const files = globby.sync("**/*", {
    cwd: testFolder,
    nodir: true,
    dot: true,
  });

  const assetsBundlerFiles = files.filter(file => file.startsWith("phenomic"));
  expect(assetsBundlerFiles.length).toBe(2);
  expect(
    fs.readFileSync(
      path.join(
        __dirname,
        "..",
        "dist",
        assetsBundlerFiles.filter(file => file.endsWith(".css")).shift(),
      ),
      { encoding: "utf8" },
    ),
  ).toMatchSnapshot();
});
