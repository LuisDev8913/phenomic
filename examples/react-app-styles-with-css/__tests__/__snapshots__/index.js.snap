// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should build example correctly 1`] = `
".helloWorld {
  font-size: 40rem
}
"
`;
